### PR TITLE
Themes: add light | dark theme

### PR DESCRIPTION
**Pull request by kseniaSs**
Added hook for changing theme. Added background variables as examples of using theme.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -5,11 +5,22 @@
 </template>
 
 <script lang="ts">
+import { useColorMode } from '@vueuse/core'
+
+const mode = useColorMode({
+  emitAuto: true
```
Comment (neSpecc) [inline src/App.vue:11]: Looks like `emitAuto` is deprecated: https://github.com/vueuse/vueuse/pull/2980/files#diff-5d628cd173c95cd5c9a1a70e3299bf4d58cceb8bcc22a31baf0cf7793f9db740R82

File: src/presentation/styles/colors.pcss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+:global(.light) {
+  --background: #fff;
+}
```
Comment (neSpecc) [inline src/presentation/styles/colors.pcss:3]: When document won't have a `.light` or `.dark` class, there will be no variable values specified? Let's make `dark` values be specified by default

File: src/presentation/styles/global.pcss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+:global(body) {
```
Comment (neSpecc) [inline src/presentation/styles/global.pcss:1]: maybe we should use [normalize.css](https://necolas.github.io/normalize.css/) instead?

File: src/App.vue [hunk 4]
```diff
@@ -5,11 +5,21 @@
 </template>
 
 <script lang="ts">
+import { useColorMode } from '@vueuse/core'
+
+const mode = useColorMode({
+  initialValue: "dark"
```
Comment (slaveeks) [inline src/App.vue:11]: Может сразу `auto` поставим?

File: src/App.vue [hunk 5]
```diff
@@ -5,11 +5,21 @@
...
+})
+
 export default {
   name: "App",
 }
 </script>
 
 <style scoped>
+  @import "./presentation/styles/colors.pcss";
```
Comment (slaveeks) [inline src/App.vue:20]: Can we import it in `index.ts` to omit it in components?

**Comment by neSpecc** [comment]
Please, update the branch from `main`. Color variables were added

**Comment by kseniaSs** [comment]
> Please, update the branch from `main`. Color variables were added

did it

**Pull request closed without merging**